.github/workflows/test.yml

```diff
@@ -16,7 +16,8 @@ on:
     - 'docs/**'
 
 permissions:
-  contents: read
+  # clean needs write permissions
+  contents: write
```

Comment:
Do you mean `build.bat clean`? Will it be possible to do something to read only as this workflow will run on PRs from forked repositories?

Reply:
These permissions are only for `GITHUB_TOKEN` 

https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

The contents suite of API's being: https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#contents

And have nothing to do with write access on the job's machine. indead `build.bat clean` has the following error warnings:

```
C:\Users\runneradmin\AppData\Local\Microsoft\dotnet\sdk\6.0.405\Microsoft.Common.CurrentVersion.targets(5599,5): warning MSB3061: Unable to delete file "D:\a\ecs-dotnet\ecs-dotnet\build\scripts\bin\Debug\net6.0\scripts.dll". Access to the path 'D:\a\ecs-dotnet\ecs-dotnet\build\scripts\bin\Debug\net6.0\scripts.dll' is denied. [D:\a\ecs-dotnet\ecs-dotnet\build\scripts\scripts.fsproj]
```

But since these do not affect the build will ignore those for now and return the read permissions.

Reply:
Note that note is more for my own sake :laughing: